lib/magma/commands.rb

```diff
@@ -2,6 +2,50 @@
 require 'logger'
 
 class Magma
+  class LoadProject < Etna::Command
+    usage '[project_name, path/to/file.json] # Import attributes into database for given project name from JSON file'
+
+    ATTRIBUTES_OPTIONS = [
+      'attribute_name',
+      'description',
+      'display_name',
+      'format_hint',
+      'created_at',
+      'updated_at',
+      'link_model_name',
+      'type',
+      'hidden',
+      'read_only',
+      'unique',
+      'index',
+      'restricted'
+    ]
+
+    def execute(project_name, file_name)
+      file = File.open(file_name)
+      file_data = JSON.parse(file.read)
+
```

Comment:
I like: `JSON.parse(File.read(file_name), symbolize_names: true)` - which also saves you from using strings below.